src/docs/patterns/navigation.mdx

```diff
@@ -22,20 +37,23 @@ Navigation is the way in which users access the different areas of your applicat
 
 ## Fixed Navigation
 
-The most basic navigation panel is fixed in place and always visible on the left side of the application. Note: a fixed navigation panel is not suitable when going to smaller resolutions or mobile devices.
+The most basic navigation panel is fixed in place and always visible on the left side of the application The navigation can be a fixed list of icons with labels or a fixed panel that is open and revealing multiple navigation options Due to internationalization and accessibility standards PX Blue recommends the use of an open panel displaying the navigation options Another benefit of using a fixed navigation is the ability to use expandable tree navigation as well as more area for product branding. Note: A fixed navigation panel is not suitable when the application is viewed on smaller resolutions or mobile devices
```

Comment:
I'm not exactly clear on what 'fixed icons with labels' and 'fixed panel' are...are we describing the Rail and the Permanent Drawer? We should match up the language.

>Do the NOTE as a callout